client/app/components/Form/index.jsx

```diff
@@ -128,14 +126,14 @@ export class Form extends React.Component<Props, State> {
     );
   };
 
-  displayQuickCreate = (input: QuickCreateProps) => {
+  displayQuickCreate = (input: any) => {
     const {
       id, name, label, placeholder, checkboxes, formProps,
     } = input;
     if (!checkboxes || !name || !label) return null;
     return (
       <div key={id}>
-        <QuickCreate
+        <Checkbox
```

Comment:
It looks like there are legitimate failing tests! I believe it's because you replaced `QuickCreate` with `Checkbox`. This component needs to be `QuickCreate` - curious why you swapped it.

Reply:
because from my understanding the final refactor involves all three of the remaining files listed under the original issue:
**form (index.jsx)**
**dynamic form**
and **QuickCreate (index.jsx)**

index.jsx form

requires in quickCreate
->

dynamicForm
requires form

quickCreate
requires dynamicForm

Reply:
So QuickCreate should be there instead of Checkbox, it's a custom component that utilizes Checkbox. I think we'll want to extract this method into another file to resolve your cyclical imports issue. Maybe put it in `ifme/client/app/components/Form/QuickCreate.jsx`?

Reply:
Hi,

I wanted to confirm for this.
What you want me to do is under *form index.jsx*

move this function
```
displayQuickCreate = (input: QuickCreateProps)
```
into it's own file under Form/QuickCreate.jsx?

Reply:
okay, updated the pr to most recent updates. 

I've moved displayQuickCreate to it's own page.

This has moved the error to be between
**index.jsx form** & **index.jsx quickCreate**

I'm trying the same idea of moving 
```
displayQuickCreateForm function

```

to a separate page from under **index.jsx quickCreate**

currently getting the following error:

```
Cannot get this.props because property props is missing in global object [1].

      4│ import type { Props as QuickCreateProps } from './index';
      5│
      6│ export const displayQuickCreateForm = (nameValue: string) => {
 [1]  7│     const { formProps } = this.props;
```
I'll continue to investigate this week!